Userland/Services/AudioServer/ClientConnection.cpp

```diff
@@ -149,4 +149,18 @@ void ClientConnection::set_main_mix_muted(bool muted)
 {
     m_mixer.set_muted(muted);
 }
+
+Messages::AudioServer::IsSelfMutedResponse ClientConnection::is_self_muted()
+{
+    if (m_queue)
+        return m_queue->is_muted();
```

Comment:
(1/2) You add `ClientAudioStream::is_muted` in the next commit, reorder these commits. :^)

Reply:
Fixed (: !